src/marqo/core/unstructured_vespa_index/constants.py

```diff
@@ -0,0 +1,7 @@
+# The threshold for a string to be considered short, long in unstructured Vespa.
+# Users will be able to filter on short strings, but not on long strings.
+SHORT_STRING_THRESHOLD = 20
+
+UNSUPPORTED_FIELD_NAME_LIST = ["::"]
+
+FILTER_STRING_BOOL_VALUE = ["true", "false"]
```

Comment:
This is only used in unstructured vespa index. It's best to keep constants local to the module if they're not shared by any other module. Helps with readability and cohesion